addons/survey/static/tests/components/set_flag_fields_tests.js

```diff
@@ -0,0 +1,205 @@
+import { click, editInput, getFixture, nextTick } from "@web/../tests/helpers/utils";
+import { makeView, setupViewRegistries } from "@web/../tests/views/helpers";
+
+
+QUnit.module("SetFlagFields", (hooks) => {
```

Comment:
Naming to update according to final widget names (and maybe split into two files, one for each field ?) Note that we could keep a single test to avoid duplicating most of the testing code.